src/NetAnalyzers/VisualBasic/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/BasicOverrideEqualsOnOverloadingOperatorEquals.vb

```diff
@@ -63,8 +63,9 @@ Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.ApiDesignGuidelines
         Private Shared Function HasEqualityOperator(type As INamedTypeSymbol) As Boolean
             For Each member In type.GetMembers()
                 Dim method = TryCast(member, IMethodSymbol)
-                If method?.MethodKind = MethodKind.UserDefinedOperator AndAlso
-                    CaseInsensitiveComparison.Equals(method.Name, WellKnownMemberNames.EqualityOperatorName) Then
+                If method IsNot Nothing AndAlso
+                   method.MethodKind = MethodKind.UserDefinedOperator AndAlso
+                   CaseInsensitiveComparison.Equals(method.Name, WellKnownMemberNames.EqualityOperatorName) Then
```

Comment:
Do we actually need case insensitive comparison? I know VB is case insensitive, but in this case the compiler should always emit `op_Equality` in the correct casing, and symbol will have its `Name` with the correct casing as well. I was wondering if this can be simplified by doing `type.GetMembers(WellKnownMemberNames.EqualityOperatorName)` and eliminate the `Name` check completely

Reply:
Good point - I have simplified the check